src/super_gradients/examples/deci_lab_export_example/deci_lab_export_example.py

```diff
@@ -4,6 +4,8 @@
 The main purpose of this code is to demonstrate how to upload the model to the platform, optimize and download it
  after training is complete, using DeciPlatformCallback.
 """
+from charset_normalizer import models
```

Comment:
import **what?**

Reply:
fixed in https://github.com/Deci-AI/super-gradients/pull/364/commits/b1b57b1e73601102855d9e3b9c31691489cb98b2